### PR TITLE
DOC-2946

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/DOC-2946
* https://harness.atlassian.net/browse/DOC-2954

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [* ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by douglas-j-bothwell** [comment]
**[DOC-2946](https://harness.atlassian.net/browse/DOC-2946)**
Previously (ie before this DOC-2946), each STO Scanner Reference topic had a “generic” screenshot of an Security step for an Aqua Trivy scan and a “generic” YAML pipeline that include steps for Trivy, Bandit, and ZAP. Folks have been commenting on Slack that this is confusing.
Given this, I’ve updated screenshots and YAML pipelines for a few scanners and removed the “generic” references for scanners where I can’t add screenshots and pipelines just now. 

**[DOC-2954](https://harness.atlassian.net/browse/DOC-2954)**
Corrected the format for invoking secrets in reusable snippets for scanner-reference topics.

File: docs/security-testing-orchestration/sto-techref-category/bandit-scanner-reference.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You can scan your container images using [Bandit](https://github.com/PyCQA/bandi
 
 ## Bandit step configuration
 
-The recommended workflow is to add an Bandit step to a Security Tests or CI Build stage and then configure it as described below. You can also configure Banditscans programmatically by copying, pasting, and editing the [YAML definition](#yaml-configuration). 
+The recommended workflow is to add an Bandit step to a Security Tests or CI Build stage and then configure it as described below. You can also configure Bandit scans programmatically by copying, pasting, and editing the [YAML definition](#yaml-configuration).
```
Comment (SudheendraKatte) [inline docs/security-testing-orchestration/sto-techref-category/bandit-scanner-reference.md:18]:
```suggestion
The recommended workflow is to add a Bandit step to a Security Tests or CI Build stage and then configure it as described below. You can also configure Bandit scans programmatically by copying, pasting, and editing the [YAML definition](#yaml-configuration). 
```

File: docs/security-testing-orchestration/sto-techref-category/snyk-scanner-reference.md [hunk 2]
```diff
@@ -22,14 +22,18 @@ import StoCreateDinD from './shared/dind-bg-step.md';
 
 ## Snyk step configuration
 
-The recommended workflow is add a Snyk step to a Security Tests or CI Build stage and then configure it as described below. You can also configure scans programmatically by copying, pasting, and editing the [YAML definition](#yaml-configuration). 
+The recommended workflow is add a Snyk step to a Security Tests or CI Build stage and then configure it as described below.
```
Comment (SudheendraKatte) [inline docs/security-testing-orchestration/sto-techref-category/snyk-scanner-reference.md:25]: Suggestion:

Recommended workflow: Add a Snyk step to a Security Tests or CI Build stage and then configure it as described below.

**Review by SudheendraKatte — APPROVED** [review]
Approved with minor comments

**Pull request closed without merging**